src/gui/network.go

```diff
@@ -83,3 +83,17 @@ func exchgConnectionsHandler(gateway Gatewayer) http.HandlerFunc {
 		wh.SendJSONOr500(logger, w, conns)
 	}
 }
+
+func defaultStatusHandler(gateway Gatewayer) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		if r.Method != http.MethodGet {
+			wh.Error405(w)
+			return
+		}
+
+		resp := gateway.GetDefaultStatus()
+
+		wh.SendJSONOr500(logger, w, &resp)
```

Comment:
It is not necessary to use `&resp` here, `resp` is fine.  It doesn't matter if the `resp` object is a pointer or not.